### DISCUSSION TITLE
frontendbr, por quê?

### DISCUSSION THREAD

**Discussion by ojonathanleal-zz** [frontendbr/forum#1029] [Discussão]
Dae galera.

Vejo que essa comunidade ajuda muita gente em ambas áreas de desenvolvimento ( front, back, devops, etc.. ).

Sempre me pergunto por que frontendbr? Se falamos de tudo um pouco, não poderiamos então mudar/criar algo que atenda o desenvolvimento de software em geral?

Obs: não é uma crítica, nem algum desmerecimento, é só um pensamento :D

**Comment by woliveiras** [comment]
Bom ponto, @ojonathanleal. :metal: 

Acredito que rolam essas conversas diversificadas por causa do Node.js...

A galera vem pelo JavaScript falando de infra, back, etc.

Uma comunidade como o frontendbr poderia fácil ser uma comunidade generalista muito fera, só seria um trampo de manutenção maior, daí teria que ver qual o objetivo com a comunidade que existe hoje: se é focar em frontend ou generalizar para ajudar qualquer público.

Hoje eu percebo que é pra ajudar quem trabalha com frontend, apesar das conversas diversas, viu. :)

**Comment by felipefialho** [comment]
@ojonathanleal Sua dúvida é muito pertinente e conforme essa organização foi crescendo e criando forma cheguei a refletir sobre esses pontos que você colocou, já que no fim, desenvolvimento é desenvolvimento né? 

Existem muitas dúvidas e discussões feitas aqui, que poderiam se extender para qualquer área de desenvolvimento, não tendo ligação efetiva com Front-end.

O @woliveiras colocou muito bem com relação ao Node. Ele de fato causa discussões mais amplas, já que é uma ferramenta usada por back e por front.

A questão é o nicho, apesar de ter assuntos em comum, existem muitos pormenores envolvendo desenvolvimento que tende a assustar quem está querendo focar em determinada tecnologia ou área. Imagina iniciantes caindo aqui e vendo um número ainda maior de linguagens sendo discutidas?

Além disso, existem outros repositórios que foram surgindo, como **vagas**, **eventos** e **me-contrata** que tendem a ser mais efetivos quando existe um nicho especifico.

Mas de verdade, gosto quando aparecem discussões amplas por aqui, até porque acho que developers precisam ter uma visão mais ampla e menos focada apenas na sua área de atuação.

**Comment by ojonathanleal-zz** [comment]
certo @woliveiras e @LFeh, concordo. Porém, acredito que os iniciantes que partem daqui, já tiveram algum embasamento para decidirem ir para o Front-end. Então fico pensando se não seria uma boa ter alguma porta de entrada para o pessoal que ainda não tem ideia do que gosta dentro da área de TI.

**Comment by kvnol** [comment]
Eu concordo com muitos pontos, mas como o @LFeh disse, é bom segmentar também. Tanto que o frontendbr inspirou o backend-br: https://github.com/backend-br/ e diversos outros fóruns que existem aqui no GitHub.

**Comment by alinebastos** [comment]
Uma boa porta de entrada é o [Training center](https://github.com/training-center/sobre)!
É uma comunidade pra TODOS iniciantes na área, pra quem tá trocando de área, quem quer avançar na carreira, etc.
Backend, frontend, devops, design, games, e o que mais tiver a ver com TI!

**Comment by ojonathanleal-zz** [comment]
Concordo com você @alinebastos. Muito bom ler as diferentes opiniões. Obrigado pessoal.
